RPG Wizardry/Assets/Scripts/ResearchData/ControlFragment.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace nl.SWEG.RPGWizardry.ResearchData
+{
+
+    public class ControlFragment : Fragment
+    {
+        #region Methods
+        #region Public
+        /// <summary>
+        /// The position which is the solution to this control fragment.
+        /// </summary>
+        public float CorrectPosition { get; set; }
+        /// <summary>
+        /// Used to determine if this fragment is solved or not.
+        /// </summary>
+        public bool Solved { get; private set; }
+        /// <summary>
+        /// the offset from the correct position which will count as a valid solution
+        /// </summary>
+        #endregion
+        #region Private
+        private float range;
```

Comment:
Add summaries

Reply:
Was already there somehow got split by the regions